pipelines/deployer/deployer.yaml

```diff
@@ -551,6 +573,24 @@ resources:
       aws_account_role_arn: ((account-role-arn))
       github_client_id: ((github-client-id))
       github_client_secret: ((github-client-secret))
+- name: namespace-role-state
+  type: terraform
+  source:
+    env_name: ((account-name))
+    backend_type: s3
+    backend_config:
+      bucket: cd-gsp-private-qndvvc
```

Comment:
Will this ever change?

Reply:
This is an artefact of the shared concourse. Can be obtained from the `info` pipeline. In theory, should not change.